docs/releases/1.10.txt

```diff
@@ -734,6 +734,9 @@ Miscellaneous
   model, you must convert them to attributes or properties as described in
   :ref:`the deprecation note <user-is-auth-anon-deprecation>`.
 
+* In the admin templates, <p class="help"> was replaced with div tag for the
```

Comment:
Add double backticks around HTML stuff so it renders as "code".
"with a div tag to allow including lists inside help messages"